packages/ember-data/lib/adapters/rest-adapter.js

```diff
@@ -838,3 +838,15 @@ function endsWith(string, suffix) {
     return string.endsWith(suffix);
   }
 }
+
+if (Ember.platform.hasPropertyAccessors) {
+  Ember.defineProperty(RestAdapter, 'maxUrlLength', {
+    enumerable: false,
+    get: function() {
+      Ember.deprecate('maxUrlLength has been deprecated (wrong casing). You should use maxURLLength instead.');
+      return this.maxURLLength;
+    }
```

Comment:
Ok, I added it at first, but thought it was a kind of read only property as this seems to me it was a static integer.